README.md

```diff
@@ -120,13 +145,16 @@ intricacies of this command:
 poetry add [PACKAGE-NAME]
 ```
 
-### Nox (Automated Testing & Other Code Changes)
```

Comment:
It's difficult to know for sure what issue you might be facing, but this is an interesting issue, I wonder if the bit about adding `--without-hashes` to the `poetry export` commands might work: https://github.com/python-poetry/poetry/issues/3472#issuecomment-744356551 

What do you mean by pinning versions in the functions in `noxfile.py`?

Reply:
Adding `--without-hashes` [here](https://github.com/ral-facilities/datagateway-api/blob/feature/add-icat-backend-documentation-%23190/noxfile.py#L17) works like you said (I also noticed it's already present in [another command further down](https://github.com/ral-facilities/datagateway-api/blob/feature/add-icat-backend-documentation-%23190/noxfile.py#L77)).

I had thought another way round it would be to include the hashes in the `pip install` command which as far as I can tell is generated [here](https://github.com/ral-facilities/datagateway-api/blob/feature/add-icat-backend-documentation-%23190/noxfile.py#L49). For example replacing `"flake8"` with `"flake8==3.7.9 "`, which is what the 4th bullet point in the comment you link suggests. But since the `--without-hashes` method seems to work (and seems simpler) I guess it'd be better to go with that?

Reply:
Thanks for testing that, I've committed that change, as per commit fb3900b25334aa141c36b33efb17bc71e137c043. Yeah that would be another way around the issue but I guess that means managing versions within the noxfile and within Poetry - not the end of the world since they're dev dependencies, but still nice to avoid. Hopefully this is now resolved.